spring-integration-core/src/main/java/org/springframework/integration/support/management/AbstractMessageChannelMetrics.java

```diff
@@ -124,6 +134,24 @@ public Counter getErrorCounter() {
 		return this.errorCounter;
 	}
 
+	/**
+	 * Return the receive counter if Micrometer metrics are being used.
+	 * @return the counter or null if Micrometer is not being used.
+	 * @since 5.0.2
+	 */
+	public Counter getReceiveCounter() {
```

Comment:
`@Nullable` as well?
At least that is what I see on all other Micrometer-related getters here...